active/progress/ProgressRing.md

```diff
@@ -0,0 +1,170 @@
+
+ProgressRing will continue to inherit from Control, and will not inherit from RangeBase. Rangebase has functionality that is not used in ProgressBar and would not be used in ProgressRing. There is an ongoing investigation into adding content (Text or icons) in the center of the ProgressRing to potentially align with Xbox and other Microsoft teams.
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+Below are properties being added to ProgressRing. The IsIndeterminate property of ProgressRing will default to True for back compatability.
+
+|Name | Type | Description | 
+|:--|:-:|:-:|
+| IsIndeterminate | boolean | Defaults to True. Gets or sets a value that indicates whether the progress ring reports generic progress with a repeating pattern (indeterminate progress) or reports progress based on the Value property (determinate progress) |
+| Value | double | Gets or sets the current setting of the range control, which may be coerced. | 
+| ShowPaused | boolean | Defaults to False. Gets or sets a value that indicates whether the progress bar should use visual states that communicate a Paused state to the user.|
+| ShowError | boolean | Defaults to False. Gets or sets a value that indicates whether the progress bar should use visual states that communicate an Error state to the user. |
+| Maximum | double | Defaults to 100. Gets or sets the highest possible Value of the range element. |
+| Minimum | double | Defaults to 0. Gets or sets the Minimum possible Value of the range element. |
+| ValueChanged | event | Event that occurs when the range value changes. |
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+# Appendix
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+# Open Questions
```

Comment:
I think there's an open question about the visual representation of a 98%, 99%, 100% full determinate ring given rounded corners, right?

Reply:
Good catch, I just added that open question!

Reply:
> I think there's an open question about the visual representation of a 98%, 99%, 100% full determinate ring given rounded corners, right?

@YuliKl @kathyang  I imagine what is done with these cases, would be tied to the Border/Stroke Thickness.  So with a thickness of 4, then at the smallest it would look like a 4 x 4 px circle at the 1% end - and a space of 4 at the 99% scale.